explore_amy.ipynb

```diff
@@ -182,7 +182,7 @@
    "source": [
     "my_data = app.retrieve() \n",
     "my_data = my_data.isel(simulation=0, scenario=0) \n",
-    "display(my_data) "
+    "display(my_data)"
```

Comment:
In the markdown right above (not sure how to highlight that portion), it says "Read the data into memory". Can you change that to something like "Read the raw hourly data into memory", then add into the description below "This is the raw data we will use to compute AMY in the next step." I just think it's a little confusing how it is now, with the user having two variables: `my_data` and `amy_data`, so I think it's good to emphasize the difference between the two data objects.

Reply:
Done!